.eslintrc

```diff
@@ -36,8 +36,17 @@
     "no-restricted-imports": ["error", "raven"],
     "prefer-rest-params": 2,
 
-    /* || va-enzyme custom plugin || */
-    "va-enzyme/unmount": 2,
+    /* || va custom plugin || */
+    "va/proptypes-camel-cased": 2,
+    "va/enzyme-unmount": 2,
+    "va/use-resolved-path": [
+      0,
```

Comment:
it's been disabled until all errors are fixed. However, this rule will go into the testing stage first before it's been enabled here. I kept it here to keep the structure of the params